.env

```diff
@@ -12,7 +12,9 @@ HASH_OPENSEARCH_PASSWORD=admin
 HASH_OPENSEARCH_PORT=9200
 HASH_OPENSEARCH_USERNAME=admin
 
-HASH_GRAPH_API_BASE_URL=http://localhost:4000
+HASH_GRAPH_API_HOST=localhost
+HASH_GRAPH_API_PORT=4000
```

Comment:
Was the URL splitting done for us to use `waitOn` for the Graph API?

Reply:
Yes for this reason, and also because this is more consistent with how we connect to other services such as open search, postgres, etc.

Also we'll be able to make use of `HASH_GRAPH_API_PORT` in the docker-compose when we have the Dockerfile for the Graph API.